src/profile-logic/tracks.js

```diff
@@ -266,6 +393,30 @@ export function computeLocalTracksByPid(
       // This thread has IPC markers.
       tracks.push({ type: 'ipc', threadIndex });
     }
+
+    const timelineAwareMarkerConsts = new Set(
+      timelineAwareMarkers.map((marker) =>
+        thread.stringTable.indexForString(marker.name)
```

Comment:
Also please note that the marker schema name is not just for the marker name. It might be the case from your importer but that's not generally true. See `getMarkerSchemaName` in `src/profile-logic/marker-schema.js` for more information. I think you'll have to implement something slightly more complex here and in other places.